cluster/rpc.go

```diff
@@ -175,14 +182,15 @@ func (w *WriteShardRequest) UnmarshalBinary(buf []byte) error {
 func (w *WriteShardRequest) unmarshalPoints() []models.Point {
 	points := make([]models.Point, len(w.pb.GetPoints()))
 	for i, p := range w.pb.GetPoints() {
-		pt, err := models.ParsePoints(p)
+		pt, err := models.NewPointFromBytes(p)
```

Comment:
Should this function be called `models.ParsePointBytes()`? `New` seems like it wouldn't involve parsing input.